astacus/common/cassandra/schema.py

```diff
@@ -108,7 +108,7 @@ def get_create_statement(cls, metadata: cm.Function) -> str:
         keyspace = cm.protect_name(metadata.keyspace)
         name = cm.protect_name(metadata.name)
         arg_list = ", ".join(
-            "{} {}".format(cm.protect_name(arg_name), arg_type)
+            f"{cm.protect_name(arg_name)} {arg_type}"
```

Comment:
(megacorp policy) no functions in f-strings please.

Reply:
Untangling that